pkg/kapp/resources/identified_resources_pods.go

```diff
@@ -35,31 +37,33 @@ func (w UniquePodWatcher) Watch(podsToWatchCh chan corev1.Pod, cancelCh chan str
 		// Watch Pods in all namespaces first and fallback to the
 		// fallbackAllowedNamespaces if lack of permission
 		namespace := ""
+		var index int
+		var forbiddenNamespaces []string
 		for {
```

Comment:
I am thinking we could do something like :

```golang
namespaces := []string{}
namespaces = append(namespaces, ...fallbackAllowedNamespaces)
for _, ns := namespaces {
// .... truncated
// And then have a break condition like this right on the err returned by Watch()
    if err ==nil && ns == "" {
        break
    }
```

Reply:
+1

Reply:
Makes sense. Done.